bitnami/nginx/templates/NOTES.txt

```diff
@@ -6,7 +6,7 @@ APP VERSION: {{ .Chart.AppVersion }}
 
 NGINX can be accessed through the following DNS name from within your cluster:
 
-    {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} (port {{ .Values.service.port }})
+    {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} (port {{ .Values.service.ports.http }})
```

Comment:
please add in README.md file information about `diagnosticMode`:

https://github.com/bitnami/charts/blob/master/template/CHART_NAME/templates/NOTES.txt#L7-L25

Reply:
Updated VIB runtime parameters and NOTES.txt